polyfill/lib/timezone.mjs

```diff
@@ -4,6 +4,8 @@ import { IDENTIFIER, EPOCHNANOSECONDS, CreateSlots, GetSlot, SetSlot } from './s
 import { ZONES } from './zones.mjs';
 import { timezone as STRING } from './regex.mjs';
 
+import bigInt from 'big-integer';
```

Comment:
why wasn't this failing? Let me guess, no tz tests?

Reply:
This was actually not a problem in the existing code, because bigInt wasn't used in this file before I made these changes. But since the return value of `GetEpochFromParts` changed to return a bigInt (see comment below) we now have to import this to do math on it.